src/components/BounceDetailsContainer/index.jsx

```diff
@@ -92,14 +94,26 @@ const BounceRuleDetailed = ({
           <Column className="details-button-column" width={1} offset={11}>
             <span>
               <Button
-                onClick={handleEditClicked}
+                onClick={
+                  editText === "FREE" || editText === "ALREADY"
```

Comment:
Since `FREE` and `ALREADY` are reused, let's store these in variables to avoid typos. `onClick` and `onKeyDown` are using the same code, let's also put that in a variable for reusability.